src/Standards/PSR12/Docs/Functions/ReturnTypeDeclarationStandard.xml

```diff
@@ -0,0 +1,39 @@
+<documentation title="Return Type Declaration">
+    <standard>
+    <![CDATA[
+    For functions and closures, the return type declaration must be correctly defined.
```

Comment:
As a standards description, this feels quite _non-descript_.

What about using the [PSR12 phrasing](https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments) (phrase just above the third code sample in the section) ?

Reply:
I will update the description 👍🏼 This does feel a bit vague 😅